azure_devops.md

```diff
@@ -0,0 +1,229 @@
+# Continuous Delivery Workshop Instructions
+
+## Import the repository into Azure DevOps
+
+Along with those in your breakout room you will have access to an ADO "project", so at this point sign into ADO at <https://aex.dev.azure.com/> and check that you can locate that. If you don't have your Azure credentials, ask a trainer at this stage.
```

Comment:
Org didn't show up quick, so offer this link as well: <https://dev.azure.com/CorndelDevOpsPipelinesWorkshop/>